multi-datacenter/read-topics.sh

```diff
@@ -15,9 +15,9 @@ docker-compose exec schema-registry-dc1 kafka-avro-console-consumer --topic topi
 echo -e "\n_schemas:"
 docker-compose exec broker-dc1 kafka-console-consumer --topic _schemas --bootstrap-server broker-dc1:29091 --from-beginning --timeout-ms 5000
 echo -e "\nprovenance info (cluster, topic, timestamp):"
-docker-compose exec connect-dc1 bash -c "export CLASSPATH=/usr/share/java/kafka-connect-replicator/connect-replicator-${CONFLUENT}.jar && kafka-console-consumer --topic topic1 --bootstrap-server broker-dc1:29091 --max-messages 10 --formatter=io.confluent.connect.replicator.tools.ProvenanceHeaderFormatter"
+docker-compose exec connect-dc1 bash -c 'JAR_P=$(ls /usr/share/java/kafka-connect-replicator/connect-replicator-*.jar) ; export CLASSPATH=${JAR_P} ; kafka-console-consumer --topic topic1 --bootstrap-server broker-dc1:29091 --max-messages 10 --formatter=io.confluent.connect.replicator.tools.ProvenanceHeaderFormatter'
```

Comment:
I tested the following if you'd prefer this simplification:

```suggestion
docker-compose exec connect-dc1 bash -c 'export CLASSPATH=/usr/share/java/kafka-connect-replicator/*; kafka-console-consumer --topic topic1 --bootstrap-server broker-dc1:29091 --max-messages 10 --formatter=io.confluent.connect.replicator.tools.ProvenanceHeaderFormatter'
```

Reply:
@rspurgeon IIRC when I tried that before, it resulted in log4j messages, which was undesirable.